src/WebJobs.Script/Workers/ProcessManagement/WorkerDescription.cs

```diff
@@ -18,6 +18,11 @@ public abstract class WorkerDescription
         /// </summary>
         public string DefaultExecutablePath { get; set; }
 
+        /// <summary>
+        /// Gets or sets the working directory for worker executable. Defaults to the Function App root directory.
+        /// </summary>
+        public string ExecutableWorkingDirectory { get; set; }
```

Comment:
To help future readers, can we add some comment about how this will be different from `WorkerDirectory` property which also exist in this class?

Reply:
In agreement. We need to make sure the description here is very clear so there's no confusion about what each one of those properties control.

Reply:
`WorkerDirectory` is the base directory of worker.

But the executables that are used to run the worker code can be anywhere in the machine, and `ExecutableWorkingDirectory` is the working directory of the worker process executing the worker, it might not be same for some of PAL scenarios, where `ExecutableWorkingDirectory` should be set to PAL executable binaries

`WorkerDirectory` doesn't update the working directory of forked worker process

---

Any suggestion on the comment is welcome, current comment came from suggestion above in the PR Comments

Reply:
@siddharth-ms  I meant to make the clarifying note/comments in the code itself (WorkerDescription class). Feel free to be more descriptive with relevant examples as our goal is to help future readers.

Reply:
Got it, added some more remarks here

Reply:
@siddharth-ms I still don’t see any detailed comments highlighting the differences in the file, similar to the one you posted earlier. Please make sure to add them before merging the PR. This will help future readers understand the difference between these properties and when to use what. Thank you!